app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/BaseViewModel.kt

```diff
@@ -40,23 +41,33 @@ abstract class BaseViewModel : ViewModel(), KoinComponent {
     fun launchIO(block: suspend CoroutineScope.() -> Unit): Job =
         viewModelScope.launch(dispatcherProvider.IO, block = block)
 
+    /**
+     * Utility function to switch coroutine context to Main.
+     *   Useful for making UI updates from IO
+     */
     suspend fun runOnMain(block: suspend CoroutineScope.() -> Unit) =
         withContext(dispatcherProvider.Main, block)
 
-    ///////////////////////////////////////////////////////////////////////////
+    // /////////////////////////////////////////////////////////////////////////
     // Error handling
-    ///////////////////////////////////////////////////////////////////////////
-    suspend fun handleError(messageId: Int) {
+    // /////////////////////////////////////////////////////////////////////////
+    /**
+     * Used to display error message with standard UI pattern
+     */
+    suspend fun handleError(@StringRes messageId: Int) {
         runOnMain {
             toaster.toast(messageId)
         }
     }
 
-    suspend inline fun <T : Any> Status<T>.handlingErrors(messageID: Int , onSuccess: (T) -> Unit): Status<T> {
+    /**
+     * Used to apply default error when handling Status, and process a success block.
```

Comment:
@BottleRocket-Colin nit: Remove `,`

```suggestion
     * Used to apply default error when handling Status and process a success block.
```